File: clients/Rust/model/src/entity_type.rs
use super::*;
#[derive(Clone, Debug, PartialEq, Eq, Hash, trans::Trans)]
pub enum EntityType {
    Wall,
    House,
    BuilderBase,
    BuilderUnit,
    MeleeBase,
    MeleeUnit,
    RangedBase,
    RangedUnit,
    Resource,
    Turret,
}

File: clients/Python-PyPy/model/entity_type.py
from enum import IntEnum

class EntityType(IntEnum):
    WALL = 0
    HOUSE = 1
    BUILDER_BASE = 2
    BUILDER_UNIT = 3
    MELEE_BASE = 4
    MELEE_UNIT = 5
    RANGED_BASE = 6
    RANGED_UNIT = 7
    RESOURCE = 8
    TURRET = 9
